etc/kubernetes/helm/enterprise-gateway/templates/deployment.yaml

```
@@ -31,6 +31,9 @@ spec:
     spec:
       # Created by this chart.
       serviceAccountName: {{ .Values.deployment.serviceAccountName }}
+  {{- if .Values.deployment.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
+  {{- end }}
```

If 30 seconds is the default, I think this might be more readable adding a `terminationGracePeriodSeconds` entry to the `values.yaml` file of `30` and updating this to:
```suggestion
      terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
```
That way operators can get a better idea of what knobs can be configured w/o looking through all the files.